registry/insecure.md

```diff
@@ -50,23 +46,26 @@ environment.
 > using this along with basic authentication requires to **also** trust the certificate into the OS cert store for some versions of docker (see below)
 {:.warning}
 
-This is more secure than the insecure registry solution.  You must configure every docker daemon that wants to access your registry
+This is more secure than the insecure registry solution.
 
-1. Generate your own certificate:
+1.  Generate your own certificate:
 
-        mkdir -p certs && openssl req \
-          -newkey rsa:4096 -nodes -sha256 -keyout certs/domain.key \
-          -x509 -days 365 -out certs/domain.crt
+    ```bash
+    $ mkdir -p certs
 
-2. Be sure to use the name `myregistrydomain.com` as a CN.
+    $ openssl req \
+      -newkey rsa:4096 -nodes -sha256 -keyout certs/domain.key \
+      -x509 -days 365 -out certs/domain.crt
+    ```
 
-3. Use the result to [start your registry with TLS enabled](./deploying.md#get-a-certificate)
+    Be sure to use the name `myregistrydomain.com` as a CN.
 
-4. Instruct every docker daemon to trust that certificate.
+2. Use the result to [start your registry with TLS enabled](./deploying.md#get-a-certificate)
 
-    This is done by copying the `domain.crt` file to `/etc/docker/certs.d/myregistrydomain.com:5000/ca.crt`.
+3. Instruct every Docker daemon to trust that certificate by copying the
+   `domain.crt` file to `/etc/docker/certs.d/myregistrydomain.com:5000/ca.crt`
```

Comment:
Is this path valid for all Docker editions? I'm thinking about Docker for Mac and Windows specifically.
Can you confirm if you really need to restart Docker? I know that in past versions you had to restart Docker if you added certs to the system cert store, but in this case we're adding certs to Docker directly, so I'm not sure we need to restart.